suite-native/module-passphrase/src/screens/PassphraseFormScreen.tsx

```diff
@@ -78,11 +88,19 @@ export const PassphraseFormScreen = () => {
                                             color="textDefault"
                                             size="medium"
                                         />
-                                        <Text color="textDefault" variant="hint">
+                                        <Text
+                                            color="textDefault"
+                                            variant="hint"
+                                            style={applyStyle(cardTextStyle)}
+                                        >
                                             <Translation id="modulePassphrase.alertCard.paragraphWarning2" />
                                         </Text>
                                     </HStack>
-                                    <Button size="small">
+                                    <Button
+                                        size="small"
+                                        colorScheme="blueBold"
+                                        iconLeft="arrowLineUpRight"
+                                    >
                                         <Translation id="modulePassphrase.alertCard.button" />
```

Comment:
Tap on the button should probably lead somewhere

Reply:
Good catch. Not sure about the link right now, but let's go with this and I'll verify it with @mnuky next week.

Reply:
I added the link, but squashed 😢 Please check the diff 🥶